docs/_source/docs/stack_config.rst

```diff
@@ -66,7 +66,7 @@ to create. For more information and valid values see the `AWS Documentation`_.
 parameters
 ~~~~~~~~~~
 
-.. container:: alert alert-danger
+.. warning::
```

Comment:
what is the reason for changing this?

Reply:
I suppose the idea of an "alert alert-danger" container is to somehow alert the user about some danger. That definition does not seem to do that. At least to me, currently that container is shown as a paragraph with some line spacing issues as https://sceptre.cloudreach.com/latest/docs/stack_config.html#parameters

You are right in that this change is not strictly related to the issue of duplicate stack config parameters. Just fixing some problems in the same section.